Final_task.py

```diff
@@ -0,0 +1,34 @@
+# Напишите код, который запускается из командной строки и получает 
+# на вход путь до директории на ПК.
+# Соберите информацию о содержимом в виде объектов namedtuple.
+# Каждый объект хранит:
+# * имя файла без расширения или название каталога,
+# * расширение, если это файл,
+# * флаг каталога,
+# * название родительского каталога.
+# В процессе сбора сохраните данные в текстовый файл используя логирование.
+
+import os
+import logging
+from collections import namedtuple
+
+logging.basicConfig(filename='output.txt', level=logging.DEBUG, format='%(asctime)s %(levelname)s:%(message)s')
```

Comment:
У вас же используется только один регистратор `info`, зачем тогда уровень `DEBAG`
```suggestion
logging.basicConfig(filename='output.txt', level=logging.INFO, format='%(asctime)s %(levelname)s:%(message)s')
```